torch/csrc/utils/pybind.cpp

```diff
@@ -7,8 +7,14 @@ namespace detail {
 
 bool type_caster<c10::SymInt>::load(py::handle src, bool) {
   if (torch::is_symint(src)) {
+    auto node = src.attr("node");
```

Comment:
Can you explain what you meant by "C++ py-bound SymNode"? Is it because of this https://github.com/pytorch/pytorch/blob/46796fe5e9b74602d45927304773fdcda1c3215a/torch/csrc/jit/python/init.cpp#L1154?